packages/components/src/utils/polyfill-theme.ts

```diff
@@ -118,12 +119,18 @@ const polyfillTheme = vsCodeTheme => {
   // To make these elements look natural with the theme,
   // we infer them from the theme
 
+  const mutedForeground = withContrast(
+    uiColors.input.placeholderForeground,
+    uiColors.sideBar.background,
+    type
+  );
+
   const addedColors = {
-    mutedForeground: withContrast(
-      uiColors.input.placeholderForeground,
-      uiColors.sideBar.background,
-      type
-    ),
+    mutedForeground,
+    activityBar: {
+      selected: uiColors.sideBar.foreground,
+      inactiveForeground: mutedForeground,
+    },
```

Comment:
it's strange that the default border of the activity bar doesn't match the sidebar, probably a bug in the theme?

Reply:
naa, its normal as I see in other themes:
![Uploading Screenshot 2020-02-03 at 22.37.15.png…]()